### DISCUSSION TITLE
aldente limit charge icon (100%, plug icon and lightning icon)

### DISCUSSION THREAD

**Discussion by macubukujkun** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#530] [Q&A]
hi
I'm a foreigner, so I can't speak English well.
But I have a question.

I'm using the 2015 MacBook Pro now.
I always used Aldente for 80% limit charge.

Today, I deleted Aldante.

The order of deletion...
1. Change the limit to 100%.
2. Delete the helper.
3. Delete Aldente.

However, when the charge is 100%, the lighting symbol is still visible.
If it's 100%, you can see the plug symbol, right?

After using the Aldente app, I haven't seen the plug icon when the battery is at 100%.

I'm curious about the reason.
And I need a solution.

I want to introduce Aldente to my friends, but if there is a problem, I can't.

![tempImagebBEJyW](https://user-images.githubusercontent.com/87821873/148068819-c64c8aa7-ea39-4234-b754-6eb9a91f4313.gif)

<img width="703" alt="스크린샷 2022-01-04 오후 10 41 08" src="https://user-images.githubusercontent.com/87821873/148068908-9eeb64f3-f191-497f-a64c-21c34f992aa8.png">
<img width="551" alt="스크린샷 2022-01-04 오후 10 41 16" src="https://user-images.githubusercontent.com/87821873/148068920-b6bde04f-f3c1-40dd-9962-e6aa535b4bfb.png">

**Comment by MatthiasKerbl** [comment]
Hi @macubukujkun,

I think your MacBooks behavior is completely normal.
I was in an Apple store the other day and every MacBook which was at 100% had the lightning symbol.
The reason why you see the plug symbol when using AlDente is that it will pause charging when you reach 100%. This will change with the next update of AlDente to ensure full charging to 100%.

Cheers,
Matthias

**Reply by macubukujkun** [reply]
I think it should be changed to a plug icon when the battery reaches 100%.
Because you said there was no problem with the MacBook, but I doubt that smc has become weird.
(Does the MacBook maintain the lighting icon to charge the battery up to 103 percent?)
